extensions/amp-ad-network-adsense-impl/0.1/test/test-amp-ad-network-adsense-impl.js

```diff
@@ -132,6 +132,18 @@ describes.realWin('amp-ad-network-adsense-impl', {
       impl = new AmpAdNetworkAdsenseImpl(element);
       expect(impl.isValidElement()).to.be.true;
     });
+    it('should be valid (correct responsive ad height)', () => {
+      element.setAttribute('data-auto-format', 'rspv');
+      element.setAttribute('height', '320');
+      expect(impl.isValidElement()).to.be.true;
+    });
+    it('should be NOT valid (wrong responsive ad height)', () => {
+      element = createAdsenseImplElement(
```

Comment:
Curious as to why in one test you create a new element/impl and in another you just set attribute?  Seems setAttribute should be sufficient, no?